core/domain/subscription_services.py

```diff
@@ -105,10 +117,13 @@ def unsubscribe_from_creator(user_id, creator_id):
 
 
 def get_all_threads_subscribed_to(user_id):
-    """Returns a list with ids of all the feedback and suggestion threads to
-    which the user is subscribed.
-
-    Callers of this function should ensure that the user_id is valid.
+    """
+    WARNING: Callers of this function should ensure that the user_id is valid.
+    Args:
+        user_id: str. User ID.
+    Returns:
```

Comment:
There should be a newline above "Returns:". Please follow the format of other docstrings in the codebase.

Reply:
Done!